src/library/Main.java

```diff
@@ -15,184 +16,190 @@
+
+                    case "LI":
+                        listItems();
+                        break;
+
+                    case "FI":
+                        fixItems();
+                        break;
+
+                    case "B":
+                        borrowItem();
+                        break;
+
+                    case "R":
+                        returnItem();
+                        break;
+
+                    case "L":
+                        listCurrentLoans();
+                        break;
+
+                    case "P":
+                        payFines();
+                        break;
+
+                    case "T":
+                        incrementDate();
+                        break;
+
+                    case "Q":
+                        isFinished = true;
+                        break;
+
+                    default:
+                        output("\nInvalid option\n");
+                        break;
                 }
-                
+
                 Library.save();
-            }            
-        } catch (RuntimeException e) {
+            }
+        }
+        catch (RuntimeException e) {
             output(e);
-        }        
+        }
+
         output("\nEnded\n");
-    }    
+    }
+
 
-    
-    private static void PAY_FINES() {
-        new PayFineUI(new pAY_fINE_cONTROL()).RuN();        
+    private static void payFines() {
+        new PayFineUI(new pAY_fINE_cONTROL()).RuN();
     }
 
 
-    private static void LIST_CURRENT_LOANS() {
+    private static void listCurrentLoans() {
         output("");
         for (Loan loan : LIBRARY.listCurrentLoans()) {
             output(loan + "\n");
-        }        
+        }
     }
 
 
-
-    private static void LIST_ITEMS() {
+    private static void listItems() {
         output("");
         for (Item book : LIBRARY.listItems()) {
             output(book + "\n");
-        }        
+        }
     }
 
 
-
-    private static void LIST_PATRONS() {
+    private static void listPatrons() {
         output("");
         for (Patron member : LIBRARY.listPatrons()) {
             output(member + "\n");
-        }        
+        }
     }
 
 
-
-    private static void BORROW_ITEM() {
+    private static void borrowItem() {
         new BorrowItemUI(new BorrowItemControl()).run();
     }
 
 
-    private static void RETURN_ITEM() {
+    private static void returnItem() {
         new ReturnItemUI(new ReturnItemControl()).run();
     }
 
 
-    private static void FIX_ITEMS() {
-        new FixItemUI(new fIX_iTeM_cONTROL()).RuN();        
+    private static void fixItems() {
+        new FixItemUI(new fIX_iTeM_cONTROL()).RuN();
     }
 
 
-    private static void INCREMENT_DATE() {
+    private static void incrementDate() {
         try {
+            Date currentDate = CALENDAR.getDate();
+            // @todo Fix method invocation as parameter.
             int days = Integer.valueOf(input("Enter number of days: ")).intValue();
```

Comment:
#PR43 - D2 : I think this breaches CSG 10 about the argument list of methods must not contain other method calls?

Is .intValue() even necessary on the end here?

Maybe something like this?
String input = input("Enter number of days: ");
int days = Integer.parseInt(input);

Reply:
https://github.com/elliotmitchum/ITC205/pull/43/commits/2d861b103c9693ee18c9024a472f3d9116c49fc3